resources/views/pages-legacy/controlpanel.blade.php

```diff
@@ -102,12 +109,178 @@ function (data) {
     return false;
 }
 
-function confirmEmailChange(event) {
-    <?php if ($permissions >= Permissions::Developer): ?>
-    return confirm('Changing your email address will revoke your privileges and you will need to have them restored by staff.');
-    <?php else: ?>
-    return true;
-    <?php endif ?>
+function handleSetMotto(newMotto) {
+    $.ajax({
+        url: '{{ route('settings.profile.update') }}',
+        type: 'PUT',
+        data: { motto: newMotto },
+        success: () => {
+            showStatusSuccess('{{ __("legacy.success.change") }}');
+        }
+    });
+}
+
+function handleSetAllowComments() {
+    const newValue = document.querySelector('#userwallactive').checked;
+
+    $.ajax({
+        url: '{{ route('settings.profile.update') }}',
+        type: 'PUT',
+        contentType: 'application/json; charset=utf-8',
+        dataType: 'json',
+        data: JSON.stringify({ userWallActive: newValue }),
+        success: () => {
+            showStatusSuccess('{{ __("legacy.success.change") }}');
+        }
+    });
+}
+
+function handleResetWebApiKeyClick() {
+    if (!confirm('Are you sure you want to reset your web API key?')) {
+        return;
+    }
+
+    $.ajax({
+        url: '{{ route('settings.keys.web.destroy') }}',
+        type: 'DELETE',
+        success: () => {
+            showStatusSuccess('{{ __("legacy.success.reset") }}');
+
+            // Temporary, will be removed in the Inertia migration.
+            setTimeout(() => {
+                window.location.reload();
+            }, 1000)
+        }
+    });
+}
+
+function handleResetConnectApiKeyClick() {
+    if (!confirm('Are you sure you want to reset your Connect API key?')) {
+        return;
+    }
+
+    $.ajax({
+        url: '{{ route('settings.keys.connect.destroy') }}',
+        type: 'DELETE',
+        success: () => {
+            showStatusSuccess('{{ _("legacy.success.reset") }}');
+        }
+    });
+}
+
+function handleDeleteAllUserComments() {
+    if (!confirm('Are you sure you want to permanently delete all comment on your wall?')) {
+        return;
+    }
+
+    $.ajax({
+        url: '{{ route('user.comment.destroyAll', $userModel) }}',
+        type: 'DELETE',
+        success: () => {
+            showStatusSuccess('{{ __("legacy.success.delete") }}');
+        }
```

Comment:
This failed.
![image](https://github.com/user-attachments/assets/3932da83-fd2b-454d-bae5-f97e574d337e)

Reply:
Fixed in latest.